Jenkinsfile

```diff
@@ -15,6 +15,9 @@ def failFast = false
 
 properties([buildDiscarder(logRotator(numToKeepStr: '50', artifactNumToKeepStr: '20')), durabilityHint('PERFORMANCE_OPTIMIZED')])
 
+//TODO: Run tests on Java 10
+//TODO: Do we really need Windows for smoke tests
```

Comment:
Yes, please, keep Windows in the smoke test sequence.  Too many problems are specific to Windows (at least in my experience).  The decision to hold locks on the file system makes creates many challenges.

Reply:
For us the main problem is Windows machines capacity. But should be fine